Fixing readme docs automation bug (#1877)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
* Removed PYYAML library (using only ruamel instead).
* Fixed an issue when filtering items using the ID set in the **create-content-artifacts** command.

> * Fixed an issue where a table was generated with empty description column in the **generate_table_section** command.

Reviewer: generate_table_section is not the command (it is the function name) the command is generate-docs
```suggestion
* Fixed an issue in the **generate-docs** command where tables were generated with an empty description column.
```
